### ISSUE TITLE
PaperNumberNode's combineNumbersIfApplicableCallback

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/make-a-ten#160]
PaperNumberNode is only created it one place, and it always provides a callback. Thus why:
```js
combineNumbersIfApplicableCallback = combineNumbersIfApplicableCallback || _.noop();
```

Second of all, if that IS needed, it should be ```_.noop``` and not ```_.noop()```, since ```_.noop()``` calls it and essentially returns undefined.

Additionally, I'd lean towards the name tryToCombineNumbers (usually 'try' is used instead of 'ifapplicable', and the 'callback' portion is generally assumed if it's typed as a function which is named with a verb).

**Comment by jonathanolson** [comment]
Also, all of this essentially applies to the parameter/callback addNumberModelCallBack. We don't want "Back" capitalized, and presumably a name of 'addNumber' or 'addNumberToModel' would be preferred.

**Comment by AshrafSharf** [comment]
Fixed as part of addressing issues  #207 , #160
